processor/src/test/java/org/mapstruct/ap/internal/util/StringsTest.java

```diff
@@ -110,5 +115,42 @@ public void findMostSimilarWord() throws Exception {
         );
         assertThat( mostSimilarWord ).isEqualTo( "fullName" );
     }
+    @Test
+    public void capitalizeEnglish() {
+        Locale.setDefault(Locale.ENGLISH);
+        String international = Strings.capitalize("international");
+        assertEquals("International", international);
+    }
+
+    @Test
+    public void decapitalizeEnglish() {
+        Locale.setDefault(Locale.ENGLISH);
+        String international = Strings.decapitalize("International");
+        assertEquals("international", international);
+    }
+
+    @Test
+    public void capitalizeTurkish() {
+        Locale.setDefault(TURKEY_LOCALE);
```

Comment:
I think that we need to store the current default `Locale` in a `Before` method and then restore it after an `After` method. This will make sure that we don't "leak" this set.